### DISCUSSION TITLE
Vector internal logs redirected to file?

### DISCUSSION THREAD

**Discussion by bmallya** [vectordotdev/vector#16562] [General]
Is there a way to redirect the vector generated internal logs to a file? I am not sure if I missed this someplace.

**Comment by jszwedko** [comment]
Hi @bmallya ,

Yes you can either just capture Vector's `stderr` (the most typical way) or you can use the `internal_logs` source in a Vector config to capture logs and direct them to a `file` sink. Note that Vector will still write them to `stderr` as well.
